frontend/src/components/wardrobe/Wardrobe.component.js

```diff
@@ -0,0 +1,136 @@
+import { setCurrentScene } from "../../redux-store/scene/sceneSlice";
+import CharacterComponent from "../design-character/character.component";
+import basicClothes from "../design-character/character-sprites/basic-clothes.png";
+import floralClothes from "../design-character/character-sprites/floral.png";
+import spaghetiiClothes from "../design-character/character-sprites/spaghetti.png";
+import { changeOutfit } from "../../redux-store/user/userSlice";
+
+const products = [
+  {
+    id: 1,
+    position: 8,
+    styleName: "tshirt-dark-blue",
+    styleSrc: basicClothes,
+  },
+  {
+    id: 2,
+    position: 20,
+    styleName: "tshirt-light-blue",
+    styleSrc: basicClothes,
+  },
+  {
+    id: 3,
+    position: 24,
+    styleName: "tshirt-brown",
+    styleSrc: basicClothes,
+  },
+  {
+    id: 4,
+    position: 32,
+    styleName: "tshirt-green",
+    styleSrc: basicClothes,
+  },
+  {
+    id: 5,
+    position: 32,
+    styleName: "tshirt-green-flower",
+    styleSrc: floralClothes,
+  },
+  {
+    id: 6,
+    position: 4,
+    styleName: "tshirt-gray",
+    styleSrc: basicClothes,
+  },
+  {
+    id: 7,
+    position: 64,
+    styleName: "vest",
+    styleSrc: spaghetiiClothes,
+  },
+];
+
+const Wardrobe = () => {
+  const dispatch = useDispatch();
+  const { character, email } = useSelector((state) => state.user);
+
+  const fetchChangeOutfit = async (email, outfit) => {
+    const response = await fetch("http://localhost:8000/users/outfit/change", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        email: email,
+        outfit: outfit,
+      }),
+    });
+    if (response.status === 201) {
+      const data = await response.json();
+      console.log("outfit update", data.message);
+    } else {
+      console.log("OPPS");
+    }
+  };
+
+  const changeOutfitHandle = async (outfit) => {
+    dispatch(changeOutfit({ top: outfit }));
+    await fetchChangeOutfit(email, { top: outfit });
+  };
+
+  useEffect(() => {}, []);
+
+  return (
+    <div className="bg-white h-screen w-screen py-5">
+      <button
+        data-test="back-to-bedroom"
+        onClick={() => dispatch(setCurrentScene("BEDROOM"))}
+        className="block ml-auto bg-yellow-400 text-white font-bold py-4 px-8 rounded-full shadow-lg hover:bg-yellow-500 transition-colors duration-200"
+      >
+        ベッドルームに戻る - Back To Bedroom
+      </button>
+
+      <div className="flex px-10">
+        <div className="flex justify-center items-center border-black border-2 rounded-lg w-1/3 h-[calc(100vh-100px)] mx-auto max-w-2xl px-4 py-16 sm:px-6 sm:py-24 lg:max-w-7xl lg:px-8">
+          <CharacterComponent data={character.currentOutfit} />
+        </div>
+        <div className="w-2/3 mx-auto max-w-2xl px-4 py-16 sm:px-6 sm:py-24 lg:max-w-7xl lg:px-8">
+          <h2 className="text-xl font-bold text-gray-900">
+            Which one you want to change?
```

Comment:
English correction - "Which one do you want to change?"

Although we could change it to something else too.

Reply:
Maybe "Pick your outfit" would be better?